docs/development/debugging.md

```diff
@@ -21,7 +21,7 @@ spec:
       containerTemplate:
         env:
           - name: NUMAFLOW_DEBUG
-            value: "true" # DO NOT forget the double quotes!!!
+            value: !!str true # DO NOT forget the double quotes!!!
```

Comment:
The comment does not make sense any more.

Reply:
i thought for a while, i wasn't sure whether to remove the quotes or not because we still need the explicit `!!str`.

Reply:
Hi @whynowy and @vigith 

Based on the discussion, would you like me to remove the comments regarding the explicit use of !!str tags altogether, or is there another preferred course of action? 

I'm happy to update the pull request accordingly.

Reply:
perhaps you can comment `# please note the !!str`

Reply:
The yaml tag is not a well known feature, not many ppl know about it - to be honest, I just learned it from [th0ger](https://github.com/th0ger)'s issue. The problem we are facing is, ppl often forget to quote the boolean value that makes the yaml unmarshal fail, that is the reason we add the comment. Updating to `!!str true` might help, but it makes it harder for ppl to understand what that means if they do not know yaml tags - I believe this applies to majority of ppl, and it's not even easy to find document about it - if you try to Google `!!str`, or `"!!str"`, you will not be able to easily find valuable information.

So, I'm struggling if this is a change we should take.

Reply:
I understand there's some hesitation about using `!!str` tags in YAML files because they might be unfamiliar to some people, to  address  this, would could go with  including a short note like @vigith says as `!!str` tells the parser to treat these values as strings which also explained in [YAML node tags](https://helm.sh/docs/chart_template_guide/yaml_techniques/#scalar-types-in-yaml) tips page. 

At the same time, I see that @th0ger prefers removing the longer comments to keep files neat, so I'm willing to go with whatever we agree is the best approach here.

Reply:
My cents. The !!str is next to the NUMAFLOW_DEBUG statement, so if people is unsure what this means it's natural to search for NUMAFLOW_ DEBUG online or in your docs.
So put an importance note in the ststic web docs for NUMAFLOW_ DEBUG?

The yaml syntax was also new to me, but I understood immediately that it was a type annotation, so I wouldn't worry.
It would keep the double qoutes though as '!!str "true"'.

Reply:
Hi @vigith, @whynowy, and @th0ger,

I've made suggested changes and tried amending the commits with the required sign-offs and , but the DCO check is still failing. I've followed the usual steps to correct this issue without success. Could you please provide some assistance or guidance on how to proceed?

Thank you for your help.